pkg/telemetry/metrics_helper.go

```diff
@@ -84,3 +77,10 @@ func computeEndpointSHAForK8sContext(ctx *configtypes.Context) string {
 	ctxBytes, _ := json.Marshal(ctx)
 	return hashString(string(ctxBytes))
 }
+
+func stringValue(val interface{}) string {
+	if val == nil {
+		return ""
+	}
+	return val.(string)
+}
```

Comment:
Considering this is a generic function, it would be better to add a type check for the `interface{}` object.
Using `val.(string)` with datatype other than string might panic.

Reply:
Updated.